streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesApplicationClientV2.scala

```diff
@@ -241,4 +242,21 @@ object KubernetesApplicationClientV2 extends KubernetesClientV2Trait with Logger
       ))
   }
 
+  @throws[Throwable]
+  def shutdown(shutDownRequest: ShutDownRequest): ShutDownResponse = {
+    val name = shutDownRequest.clusterId
+    val namespace = shutDownRequest.kubernetesDeployParam.kubernetesNamespace
+
+    def richMsg: String => String = s"[flink-shutdown][clusterId=$name][namespace=$namespace] " + _
+
+    FlinkK8sOperator.k8sCrOpr.deleteDeployment(namespace, name).runIOAsTry match {
```

Comment:
If you directly use the `CROperator` internal API, you need to manually untrack the corresponding resources with `FlinkK8sObserver.untrack`.